src/webgpu/api/validation/render_pipeline/depth_stencil_state.spec.ts

```diff
@@ -205,6 +209,37 @@ g.test('depth_write,frag_depth')
     t.doCreateRenderPipelineTest(isAsync, hasDepth, descriptor);
   });
 
+g.test('depth_bias')
+  .desc(`Depth bias parameters are only valid with triangle topologies.`)
+  .params(u =>
+    u
+      .combine('isAsync', [false, true])
+      .combine('topology', kPrimitiveTopology)
+      .beginSubcases()
+      .combine('depthBias', [undefined, 0, 1])
+      .combine('depthBiasSlopeScale', [undefined, 0, 1])
+      .combine('depthBiasClamp', [undefined, 0, 1])
```

Comment:
Should test values less than 0 for these as well, just to make sure the validation is implemented correctly.

And that would make it blow up unnecessarily a bit (64 subcases) so consider something like
```ts
.combineParams([
  {},
  { depthBias: -1 },
  { depthBias: 0 },
  { depthBias: 1 },
  { depthBiasSlopeScale: -1 },
  { depthBiasSlopeScale: 0 },
  { depthBiasSlopeScale: 1 },
  { depthBiasClamp: -1 },
  { depthBiasClamp: 0 },
  { depthBiasClamp: 1 },
])
```
or equivalently (I think):
```ts
.combineParams([
  {},
  ...u.combine('depthBias', [-1, 0, 1]),
  ...u.combine('depthBiasSlopeScale', [-1, 0, 1]),
  ...u.combine('depthBiasClamp', [-1, 0, 1]),
])
```

Otherwise LGTM

Reply:
Done. Second variant you show _does_ work, FWIW!

Reply:
Done. Second variant you show _does_ work, FWIW!